controllers/custom-error-controller.js

```diff
@@ -0,0 +1,5 @@
+exports.customErr = (err, req, res, next) => {
+  if (err.status === 400 && err.msg === "article does not exist") {
```

Comment:
Again, a 404 would be better for this error as 400 implies Bad Request === bad data input/wrong data type